src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.tsx

```diff
@@ -70,6 +71,23 @@ const getTab = (key: string) => {
   }
 };
 
+const getCurrentObject = (
```

Comment:
It took me a bit of time to understand what this does, so maybe it might be a good idea to add a small comment here?

Reply:
Yes! Sorry, I definitely should have left some comments here. I've added a bunch to this function, so hopefully it makes more sense now.